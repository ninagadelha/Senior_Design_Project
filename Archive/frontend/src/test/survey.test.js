import renderer from "react-test-renderer";
import Survey from "../routes/survey";

jest.mock("react-router-dom");
jest.mock("axios");
jest.mock("../styles/survey.css");

const renderTree = (tree) => renderer.create(tree);
describe("<Survey>", () => {
  it("should render component", () => {
    expect(renderTree(<Survey />).toJSON()).toMatchSnapshot();
  });
});
