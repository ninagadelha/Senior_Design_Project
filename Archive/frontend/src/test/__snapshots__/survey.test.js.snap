// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Survey> should render component 1`] = `
[
  <nav
    className="navbar"
  >
    <div
      className="navbar__container"
    >
      <div
        className="navbar__toggle"
        id="mobile-menu"
        onClick={[Function]}
      >
        <span
          className="bar"
        />
         
        <span
          class="bar"
        />
        <span
          className="bar"
        />
      </div>
      <ul
        className="navbar__menu"
      >
        <li
          className="navbar__item"
        />
        <li
          className="navbar__item"
        />
        <li
          className="navbar__item"
        />
        <li
          className="navbar__item"
        />
      </ul>
    </div>
  </nav>,
  <div
    className="survey-page"
  >
    <div
      className="survey-card"
    >
      <h1
        className="header-survey"
      >
        Survey
      </h1>
       
      <h2
        className="header2-survey"
      >
        General Questions
      </h2>
      <div
        className="input-container"
      >
        Which program are you in?
        <select
          onChange={[Function]}
        >
          <option
            disabled={false}
            value=""
          >
            Please select a program
          </option>
        </select>
        <p
          className="error-label"
        />
      </div>
      <div
        className="input-container"
      >
        <p
          className="input-title"
        >
          What is your age?
        </p>
        <input
          className="text-input"
          onChange={[Function]}
          type="text"
        />
        <p
          className="error-label"
        />
      </div>
      <div
        className="page-one-checks"
      >
        <p
          className="input-title"
        >
          Please indicate your gender (select all that apply)
        </p>
        <label>
          <input
            checked={false}
            onChange={[Function]}
            type="checkbox"
            value=""
          />
          Woman
        </label>
        <div />
        <label>
          <input
            checked={false}
            onChange={[Function]}
            type="checkbox"
            value=""
          />
          Man
        </label>
        <div />
        <label>
          <input
            checked={false}
            onChange={[Function]}
            type="checkbox"
            value=""
          />
          Transgender
        </label>
        <div />
        <label>
          <input
            checked={false}
            onChange={[Function]}
            type="checkbox"
            value=""
          />
          Non-binary/Non-conforming
        </label>
        <div />
        <label>
          <input
            checked={false}
            onChange={[Function]}
            type="checkbox"
            value=""
          />
          Prefer not to respond
        </label>
        <p
          className="error-label"
        />
      </div>
      <div
        className="page-one-checks"
      >
        <p
          className="input-title"
        >
          Please indicate your race/ethnicity (select all that apply)
        </p>
        <label>
          <input
            checked={false}
            onChange={[Function]}
            type="checkbox"
            value=""
          />
          African American
        </label>
      </div>
      <div
        className="page-one-checks"
      >
        <label>
          <input
            checked={false}
            onChange={[Function]}
            type="checkbox"
            value=""
          />
          White
        </label>
      </div>
      <div
        className="page-one-checks"
      >
        <label>
          <input
            checked={false}
            onChange={[Function]}
            type="checkbox"
            value=""
          />
          Hispanic/Latino(a)
        </label>
      </div>
      <div
        className="page-one-checks"
      >
        <label>
          <input
            checked={false}
            onChange={[Function]}
            type="checkbox"
            value=""
          />
          Asian
        </label>
      </div>
      <div
        className="page-one-checks"
      >
        <label>
          <input
            checked={false}
            onChange={[Function]}
            type="checkbox"
            value=""
          />
          American Indian/Alaska Native
        </label>
      </div>
      <div
        className="page-one-checks"
      >
        <label>
          <input
            checked={false}
            onChange={[Function]}
            type="checkbox"
            value=""
          />
          Native Hawaiian/Other Pacific Islander
        </label>
      </div>
      <div
        className="page-one-checks"
      >
        <label>
          <input
            checked={false}
            onChange={[Function]}
            type="checkbox"
            value=""
          />
          Prefer not to respond
        </label>
        <p
          className="error-label"
        />
      </div>
      <div
        className="input-container"
      >
        <p
          className="input-title"
        >
          How many higher education credits have you taken?
        </p>
        <input
          className="text-input"
          onChange={[Function]}
          type="text"
        />
        <p
          className="error-label"
        />
      </div>
      <div
        className="input-container"
      >
        <p
          className="input-title"
        >
          What is your STEM area of interest?
        </p>
        <input
          className="text-input"
          onChange={[Function]}
          type="text"
        />
        <p
          className="error-label"
        />
      </div>
      <div
        className="input-container"
      >
        <p
          className="input-title"
        >
          Which institution of higher education are you attending (or planning on attending)?
        </p>
        <input
          className="text-input"
          onChange={[Function]}
          type="text"
        />
        <p
          className="error-label"
        />
      </div>
      <button
        className="next-button"
        onClick={[Function]}
      >
        Next
      </button>
    </div>
  </div>,
]
`;
